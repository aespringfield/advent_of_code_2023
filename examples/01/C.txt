eighteightsrfcxtvg7three1two9nineeightwolqn
6twodndmhcgxlgbqbqndbbthnngblfgtzh5fouroneightrjp
9seven453seventhree67oneightv
22trflsckfg7eighteightoneightrbk
