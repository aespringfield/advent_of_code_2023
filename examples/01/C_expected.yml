part_two: 276
